dags/data_contributions.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import airflow.operators
+from airflow.models import DAG
+import utils.helpers as helpers
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 3, 1),
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag = DAG(
+    dag_id='data_contributions',
+    default_args=args,
+    max_active_runs=1,
+    schedule_interval='@daily'
```

Comment:
So, there're two intervals: `schedule_interval` and `poke_interval`. The first controls when the DAG should run, the latter is related to the `Sensor` and controls how often it'll poke (e.g. run the SQL query and check the results).

Imagine that we have a DAG with `schedule_interval` as `@monthly`, configured to start on the first of each month. On the scheduled date, it'll start and run its tasks. The `Sensor` will be triggered and start poking for whatever it's sensing (according to its `poke_interval`). When it's successful, it'll trigger its downstream tasks. However, it won't resume poking until the next `schedule_interval`.

In a kind-of pseudocode, what's happening is:
```python
def run_sensor():
    # Wait until the sensor is successful
    while True:
        if sensor.poke():
            break
        sleep(sensor.poke_interval)

    # Trigger downstream tasks
    sensor.downstream.run()
```

This hypothetical `run_sensor()` method is called every time the DAG starts (which is controled by its `schedule_interval`).

If we want the sensor to be continuously poking, we need to add a `TriggerDagOperator` at the end of the tasks to re-trigger the DAG again.

(Thanks @JohnCulver for the explanation)